pkg/utils/get-repository-info.go

```diff
@@ -0,0 +1,57 @@
+package utils
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+)
+
+// GetRepoUrl - Gets the URl of a git repo.
+func GetRepoUrl() (string, error) {
+	gitLocation, err := exec.LookPath("git")
+
+	if err != nil {
+		return "", fmt.Errorf("unable to find git on system: %w", err)
+	}
+
+	cmd := exec.Command(gitLocation, "config", "--get", "remote.origin.url")
```

Comment:
I've reworked this  so that if it fails it will then get a list of the remotes and use the first one to get the remote url from. If it fails at any point it will just return an empty string and not error.